src/array/show.jl

```diff
@@ -88,6 +88,7 @@ function _print_matrix(io::IO, A::AbstractArray{<:Any,1}, lookups::Tuple)
     Base.print_matrix(io, A_dims)
     return nothing
 end
+_print_matrix(io::IO, A::AbstractDimArray, lookups::Tuple) = _print_matrix(io, parent(A), lookups) 
```

Comment:
What does this new method do?

Reply:
This shows the matrix with the dimension values appended. It was breaking because the `cat` was out of alignment and we check that now